website/docs/segments/argocd.mdx

```diff
@@ -0,0 +1,39 @@
+---
+id: argocd
+title: ArgoCD Context
+sidebar_label: ArgoCD
+---
+
+## What
+
+Display the current ArgoCD context name, user and/or server.
+
+## Sample Configuration
+
+import Config from '@site/src/components/Config.js';
+
+<Config data={{
+    "type": "argocd",
+    "style": "powerline",
+    "powerline_symbol": "\uE0B0",
+    "foreground": "#ffffff",
+    "background": "#FFA400",
+    "template": " {{ .Context.Name }}:{{ .Context.User}}@{{ .Context.Server }} "
+}}/>
+
+## Template ([info][templates])
+
+:::note default template
+
+```template
+{{ .Context.Name }}
+```
+### Properties
+
+| Name       | Type     | Description                  |
```

Comment:
the table needs to be formatted correctly (so all columns align).

Reply:
columns have been aligned.